apps/web/playwright/fixtures/users.ts

```diff
@@ -229,12 +232,48 @@ export const createUsersFixture = (page: Page, workerInfo: WorkerInfo) => {
                 id: _user.id,
               },
             },
-            schedulingType: "COLLECTIVE",
-            title: "Team Event - 30min",
-            slug: "team-event-30min",
+            schedulingType: scenario.schedulingType ?? SchedulingType.COLLECTIVE,
+            title: teamEventTitle,
```

Comment:
Wouldn't it be better to accept these in function arg and have default values of what they are hardcoded to currently.

We can then return these values from the function and then use those itself instead of using the imported values.